core/src/main/java/org/apache/iceberg/view/BaseMetastoreViewCatalog.java

```diff
@@ -79,6 +81,8 @@ protected BaseViewBuilder(TableIdentifier identifier) {
       Preconditions.checkArgument(
           isValidIdentifier(identifier), "Invalid view identifier: %s", identifier);
       this.identifier = identifier;
+      this.properties.putAll(
```

Comment:
we might want to do it similarly to https://github.com/apache/iceberg/blob/817a5e1be1616af77329965ac3742c14ca3ae116/core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java#L258-L270 so that we have some logging when those defaults are set